interface/global-preferences.md

```diff
@@ -0,0 +1,55 @@
+# Global Preferences 
+**Global Preferences** allows us to apply default settings for aspects of our Incari **Project**. These aspects are discussed below:
+
+{% tabs %}
+
+{% tab title="Animation Editor" %}
+![](../.gitbook/assets/interface/global-preferences/animation-editor.png)
+{% endtab %}
+
+## Playback
+### Loop Mode 
+`Loop Mode` determines if an animation should play forward or alternate between playing forward and backward in a **Scene**. It can be set in the following ways:
+* `Alternate`: The animation is played forward and backward in each animation cycle.
+* `None`: The animation is not played.
```

Comment:
Not looped! As in it plays once.

There are some animated GIFs to illustrate the looping behaviors in the assets folder:

![image](https://user-images.githubusercontent.com/20472837/91956060-65b16c80-ed04-11ea-9c4c-3c4472f67e93.png)